docs/user_guide/reporting/Supervisor Notifications

```diff
@@ -0,0 +1,22 @@
+grand_parent: User Guide
```

Comment:
Looks like this is missing a "parent:" tag. I don't think it can just have a grandparent. Look to other pages.